tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -404,30 +375,27 @@ df_rec[!, :cell] = 1:n_cells
 ```
 
 We observe that, the data in the original space is recovered with key information, the distinct feature values in the first and last three genes for the two cell groups, are preserved.
-We can also examine the data in the reduced space, i.e. the selected components (rows) in $z$.
+We can also examine the data in the dimension-reduced space, i.e. the selected components (rows) in $\mathbf{Z}$.
 
 ```julia
-df_pro = DataFrame(z[alpha_indices, :]', :auto)
+df_pro = DataFrame(Z[α_indices, :]', :auto)
 rename!(df_pro, Symbol.(["z" * string(i) for i in collect(1:k)]))
-df_pro[!, :cell] = 1:n_cells
-
 df_pro[!, :type] = repeat([1, 2]; inner=n_cells ÷ 2)
 scatter(
     df_pro[:, 1], df_pro[:, 2]; xlabel="z1", ylabel="z2", color=df_pro[:, "type"], label=""
 )
 ```
 
-```
-This plot is very similar to the low-dimensional plot above, with the *relevant* dimensions chosen based on the values of $\alpha$ via ARD. When you are in doubt about the number of dimensions to project onto, ARD might provide an answer to that question.
+This plot is very similar to the low-dimensional plot above, with the *relevant* dimensions chosen based on the values of $α$ via ARD. When you are in doubt about the number of dimensions to project onto, ARD might provide an answer to that question.
 
 ## Final comments.
 
 p-PCA is a linear map which linearly transforms the data between the original and projected spaces.
 It can also thought as a matrix factorisation method, in which $\mathbf{X}=(\mathbf{W} \times \mathbf{Z})^T$. The projection matrix can be understood as a new basis in the projected space, and $\mathbf{Z}$ are the new coordinates.
 
-References:
-1. Gilbert Strang, <Introduction to Linear Algebra>, 5th Ed., Wellesley-Cambridge Press, 2016
-2. Probabilistic PCA by TensorFlow, https://www.tensorflow.org/probability/examples/Probabilistic_PCA
-3. Gareth M. James, Daniela Witten, Trevor Hastie, Robert Tibshirani, <An Introduction to Statistical Learning>, Springer, 2013.
-4. David Wipf, Srikantan Nagarajan, A New View of Automatic Relevance Determination, NIPS 2007.
-5. Christopher Bishop, <Pattern Recognition and Machine Learning>, Springer, 2006.
+#References:
+#1. Gilbert Strang, \<Introduction to Linear Algebra\>, 5th Ed., Wellesley-Cambridge Press, 2016.
+#2. Probabilistic PCA by TensorFlow, "https://www.tensorflow.org/probability/examples/Probabilistic_PCA".
+#3. Gareth M. James, Daniela Witten, Trevor Hastie, Robert Tibshirani, \<An Introduction to Statistical Learning\>, Springer, 2013.
+#4. David Wipf, Srikantan Nagarajan, A New View of Automatic Relevance Determination, NIPS 2007.
+#5. Christopher Bishop, \<Pattern Recognition and Machine Learning\>, Springer, 2006.
```

Comment:
Why was this commented out? This could/should be made proper footnotes/references, e.g., something like
```suggestion
## References:

[^1]: Gilbert Strang, *Introduction to Linear Algebra*, 5th Ed., Wellesley-Cambridge Press, 2016.
[^2]: Probabilistic PCA by TensorFlow, "https://www.tensorflow.org/probability/examples/Probabilistic_PCA".
[^3]: Gareth M. James, Daniela Witten, Trevor Hastie, Robert Tibshirani, *An Introduction to Statistical Learning*, Springer, 2013.
[^4]: David Wipf, Srikantan Nagarajan, *A New View of Automatic Relevance Determination*, NIPS 2007.
[^5]: Christopher Bishop, *Pattern Recognition and Machine Learning*, Springer, 2006.
```